### ISSUE TITLE
Formatting fail at the end of core/howto/trial.html

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7475]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#7475|
|Type|defect|
|Created|2014-06-04 15:28:13Z|
|Branch|https://github.com/twisted/twisted/tree/docs-for-trial-7475|

At the very bottom of https://twistedmatrix.com/documents/current/core/howto/trial.html there is some weird formatting where the sentence that says "improve this document" is broken by a big chunk of whitespace before concluding with a link to twistedmatrix.com (not a very useful link either, maybe a link to `newticket` or a wiki page about filing tickets would be better).

Attachments:

* [docs-for-trial-7475-1.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ac5/ac5884647d6508629aab96d26dd4897d738db1fe/389af4f058f71f260236911d89ff3023a2a509cd.patch) (1008 bytes) - added by herrwolfe on 2014-06-05 00:02:36Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7475 7475
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_docs_for_trial_7475 branches/docs-for-trial-7475
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1401895693000000 1401895693000000
changetime__1474198049187875 1474198049187875
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)|@derwolfe set owner to @derwolfe|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)<a name="note_2"></a>|@derwolfe removed owner|
|-|-|

In docs-for-trial-7475-1.patch the docs have been edited to remove excess whitespace before the final link. I have also replaced final link to the main Twisted web page with a link to the contribution guidelines.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

(In [42779]) Branching to docs-for-trial-7475.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

Looks good to merge.  Many thanks!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [42785]) Merge docs-for-trial-7475

Author: herrwolfe
Reviewer: adiroiban
Fixes: [#7475](https://github.com/twisted/twisted/issues/7475)

Fix a formatting error in the trial howto.  Also introduce a better link
guiding folks towards filing tickets against Twisted.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
